### ISSUE TITLE
Angular 6: Unexpected token ] in JSON at position 970

### ISSUE THREAD

**Issue by Rulettaja** [angular/angular-cli#11621]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] bug report -> please search issues before submitting
- [ ] feature request
```

### Command (mark with an `x`)
```
- [ ] new
- [ ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [x ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Versions

```
Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.8
@angular-devkit/build-angular     0.6.8
@angular-devkit/build-optimizer   0.6.8
@angular-devkit/core              0.6.8
@angular-devkit/schematics        0.6.8
@angular/cli                      6.0.8
@ngtools/webpack                  6.0.8
@schematics/angular               0.6.8
@schematics/update                0.6.8
rxjs                              6.2.2
typescript                        2.7.2
webpack                           4.8.3

```
node --version: v8.10.0
npm --version: 6.2.0
Windows (10)

### Repro steps

ng g c name

### The log given by the failure

Unexpected token ] in JSON at position 970

### Desired functionality

I'd like to create Angular component.
Error just randomly occured when I started coding.

### Mention any other details that might be useful

-

**Comment by filipesilva** [comment]
Heya, can you show us your `angular.json` file please?

**Comment by Brocco** [comment]
This issue is likely resolved with PR #11132 which is included with the version 6.1.0 release.

**Comment by hansl** [comment]
Closing this as fixed with PR #11132. If you upgrade to 6.1.1 and this still happens please file a new issue passing in your JSON files.
